spec.html

```diff
@@ -18005,28 +18051,27 @@ <h2>Syntax</h2>
       <emu-clause id="sec-runtime-semantics-arrayaccumulation" oldids="sec-static-semantics-elisionwidth" type="sdo">
         <h1>
           Runtime Semantics: ArrayAccumulation (
-            _array_: unknown,
-            _nextIndex_: unknown,
-          )
+            _array_: an Array,
+            _nextIndex_: an integer,
+          ): a Completion Record normally containing an integer
         </h1>
         <dl class="header">
         </dl>
         <emu-grammar>Elision : `,`</emu-grammar>
         <emu-alg>
           1. Let _len_ be _nextIndex_ + 1.
           1. Perform ? Set(_array_, *"length"*, 𝔽(_len_), *true*).
-          1. NOTE: The above Set throws if _len_ exceeds 2<sup>32</sup>-1.
+          1. NOTE: The above step throws if _len_ exceeds 2<sup>32</sup>-1.
           1. Return _len_.
         </emu-alg>
         <emu-grammar>Elision : Elision `,`</emu-grammar>
         <emu-alg>
-          1. Return the result of performing ArrayAccumulation of |Elision| with arguments _array_ and _nextIndex_ + 1.
+          1. Return ? ArrayAccumulation of |Elision| with arguments _array_ and _nextIndex_ + 1.
```

Comment:
```suggestion
          1. Return ? ArrayAccumulation of |Elision| with arguments _array_ and (_nextIndex_ + 1).
```
Otherwise, there's an ambiguity: an alternative parse would have `? ArrayAccumulation of |Elision| with arguments _array_ and _nextIndex_` as the left-hand operand of the `+`.